packages/e2e-tests/recover/seedPhrase.spec.js

```diff
@@ -0,0 +1,48 @@
+const { test, expect } = require("@playwright/test");
+
+const { createTestSubaccount } = require("../utils/account");
+
+const { describe, beforeAll, afterAll } = test;
+
+describe("Account Recovery Using Seed Phrase", () => {
+    let testAccount;
+
+    beforeAll(async () => {
+        testAccount = await createTestSubaccount();
+    });
+
+    /** Note: afterAll hook does not currently run if tests timeout / fail:
+     * https://github.com/microsoft/playwright/pull/8008
+     */
+    afterAll(async () => {
+        await testAccount.delete();
+    });
+
+    test("navigates to seed phrase page successfully", async ({ page }) => {
+        await page.goto("/");
+
+        await page.click(
+            `button:text-matches("Import Existing Account", "i")`
+        );
+        await page.click(`a:text-matches("Recover Account", "i")`);
```

Comment:
Would prefer that we use test IDs here rather than rely on the exact wording of the text the user sees.
Imagine if we change `Recover Account` to `Recover your Account` --  we really wouldn't want our CI to fail due to a failed e2e test here in that case; the important thing is less what the text on the element is, but more what role that button fulfills.

Reply:
Done 👍